### PR TITLE
cards for DYJetsToLL_M-4to50_HT binned sample

### PR DESCRIPTION
**Pull request by anirban1982**
all crads for DYJetsToLL_M-4to50_HT binned sample

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_lowmll/DYJets_HT-100to200/DYJets_HT-low_mll_100to200_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+ 1d5   = ebmax ! maximum E for the b
+ 1d5   = eamax ! maximum E for the photons
+ 1d5   = elmax ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+   5  = etaj    ! max rap for the jets 
+  1d2  = etab    ! max rap for the b
+ 2.5  = etaa    ! max rap for the photons 
+ 1d2  = etal    ! max rap for the charged leptons
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_lowmll/DYJets_HT-100to200/DYJets_HT-low_mll_100to200_run_card.dat]: is there any specific reason to keep the  max rap for the charged leptons to 1.2 and not 2.5 ?

Comment (efeyazgan) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYJets_HT_lowmll/DYJets_HT-100to200/DYJets_HT-low_mll_100to200_run_card.dat]: same question for b quarks

**Comment by anirban1982** [comment]
changed etal and etab along with he lepton defn in proc card,

define l+ = e+ mu+ ta+
define l- = e- mu- ta-

**Comment by perrozzi** [comment]
hoping not to miss anything else, things look ok to me now
(indeed also the lepton definition is important to include taus!)

**Comment by anirban1982** [comment]
@perrozzi can you please merge in that case?

**Comment by anirban1982** [comment]
Thanks Luca @perrozzi .

**Pull request merged**